data/cookbook/text-processing/regular-expressions-ppxregexp/00-ppx_regexp.ml

```diff
@@ -6,7 +6,7 @@ packages:
   version: "0.5.1"
 libraries:
   - re
-PPXs:
+ppxes:
```

Comment:
is "ppxes" a specific command or are you talking about multiple PPX? If the latter, the style guide has this formatted as PPXs.

Reply:
Here, `ppxes` is the token expected by [cookbook.ml](https://github.com/ocaml/ocaml.org/tree/cookbook/tool/ood-gen/lib/cookbook.ml#L25)

Reply:
Thanks for clarifying!